Gemfile

```diff
@@ -20,8 +22,7 @@ gem 'decidim-selectable-news', path: 'decidim-selectable-news'
 gem 'decidim-admin-search_user', path: 'decidim-admin-search_user'
 #### Custom gems and modifciations block end ####
 
-gem 'decidim-idcat_mobil', git: "https://github.com/gencat/decidim-idcat_mobil.git", branch: "master"
-gem 'omniauth-idcat_mobil', git: "https://github.com/gencat/omniauth-idcat_mobil.git", branch: "master"
```

Comment:
Why you delete this gem?

Reply:
Because `decidim-idcat_mobil` already depends upon `omniauth-idcat_mobil`. It was declared here to force pulling from the git repo. Not it should not be necessary anymore.

Reply:
:+1: